### PR TITLE
Fix svace issue in Kernel, Task Manager, Messaging

### PR DESCRIPTION
**Pull request by jeongchanKim**

### CODE REVIEW

**Comment by seinfra** [comment]
Target : [10b2c89fa4ab24353f5f35ea632a283d72e5cf83] - Code Rule Check OK.

**Comment by seinfra** [comment]
Target : [10b2c89fa4ab24353f5f35ea632a283d72e5cf83] - Code Rule Check (C++) OK.

**Comment by seinfra** [comment]
Target : [27357dd3525acf029d295560d3833f44bfdffe31] - Code Rule Check (C++) OK.

**Comment by seinfra** [comment]
Target : [27357dd3525acf029d295560d3833f44bfdffe31] - Code Rule Check OK.

**Comment by seinfra** [comment]
Target : [7e11bc6eaa31e7517305d0638665fdab10ba1b7e] - Code Rule Check OK.

**Comment by seinfra** [comment]
Target : [7e11bc6eaa31e7517305d0638665fdab10ba1b7e] - Code Rule Check (C++) OK.

**Comment by seinfra** [comment]
Target : [b1d92da85bd31003b3d021414dd5af24513adc08] - Code Rule Check (C++) OK.

**Comment by seinfra** [comment]
Target : [b1d92da85bd31003b3d021414dd5af24513adc08] - Code Rule Check OK.

**Comment by seinfra** [comment]
Target : [99231150a29dfb83590dcbe1d7d0c7f915a77cdb] - Code Rule Check (C++) OK.

**Comment by seinfra** [comment]
Target : [99231150a29dfb83590dcbe1d7d0c7f915a77cdb] - Code Rule Check OK.

File: framework/src/task_manager/task_manager_set_callback.c [hunk 1]
```diff
@@ -293,15 +293,19 @@ int task_manager_set_broadcast_cb(int msg, tm_broadcast_callback_t func, tm_msg_
 
 	TM_ASPRINTF(&request_msg.q_name, "%s%d", TM_PRIVATE_MQ, request_msg.caller_pid);
 	if (request_msg.q_name == NULL) {
-		TM_FREE(((tm_msg_t *)((tm_broadcast_info_t *)request_msg.data)->cb_data)->msg);
-		TM_FREE(((tm_broadcast_info_t *)request_msg.data)->cb_data);
+		if (((tm_broadcast_info_t *)request_msg.data)->cb_data != NULL) {
+			TM_FREE(((tm_msg_t *)((tm_broadcast_info_t *)request_msg.data)->cb_data)->msg);
```
Comment (sijupunnnoose) [inline framework/src/task_manager/task_manager_set_callback.c]: If TM_FREE macro can have the null check we dont need to add this check everywhere.

Comment (jeongchanKim) [reply framework/src/task_manager/task_manager_set_callback.c]: That's a good idea. I will modify this with another PR.
Thanks!

**Pull request merged**